Sources/ApolloCodegenLib/Templates/ImportStatementTemplate.swift

```diff
@@ -1,20 +1,21 @@
+import ApolloUtils
+
+/// Provides the format to import Swift modules required by the template type.
```

Comment:
Note that both `ImportStatementTemplate` (this) and `HeaderCommentTemplate` don't conform to `TemplateRenderer`. They're not 'traditional' templates since they're never output to a file alone, they get pulled into other templates.

I considered making them private in `TemplateRenderer.swift` to avoid confusion about why they don't conform to the protocol but that complicates testing. I'm open to suggestions on how to better structure that.

Reply:
That makes sense to me!

Reply:
Done in 77fc9028dc72a00db3dcf3a8dafa23faa1dd45da.